test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void noInputShouldReturnFalse() {
```

Comment:
This is not the standard to name the test case method. It should be methodTesting_ScenarioTesting_ExpectedReturnValue.

i.e. isAnyNull_emptyList_falseReturned